### DISCUSSION TITLE
[Question]: Librechat.yaml not working?

### DISCUSSION THREAD

**Discussion by Aunchers** [danny-avila/LibreChat#1499] [Troubleshooting] [answered]
### What is your question?

(I would like to specify, I do not believe this is a bug. I solely believe that I configured wrong)
So, I wanted to add a new api to Librechat.
I edited the .yaml file to include my new api, and the setup acknowledged that there was custom settings and "loaded" the custom settings. Now the API list includes Mistral and Openrouter, but my api is nowhere to be seen.

The below image is my api in the .yaml file (the mistral, and openrouter specifications are there. I just did not include them in the image)
![image](https://github.com/danny-avila/LibreChat/assets/91509158/a90672e8-ba84-4e42-b9f8-4ddf979a1dcb)

This image is my model list on the frontend
![image](https://github.com/danny-avila/LibreChat/assets/91509158/fd3212be-94a4-4c4e-946c-0d631274d242)

### More Details

**[Extra info]**
I am running the latest (as of writing) build on a windows 11 laptop with Docker.
I followed the installation exactly and have only changed the .yaml and the .env ,
The only things added to the .env was disabling email login/registration 

### What is the main subject of your question?

Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
If you’re using docker, you may need to rebuild after making changes to the file.

The docker setup for this will change as currently it’s being required, and setting up the file should be optional as it is without docker.

Also if you are using Redis, you would need to flush the cache. There is no option to flush the cache from the app yet, but you can omit it to test it if you are

**Comment by danny-avila** [comment]
I just made the necessary update to make `librechat.yaml` optional as expected: https://docs.librechat.ai/install/configuration/custom_config.html#docker-setup

This should also ensure it works for you

**Comment by Aunchers** [comment, marked as answer]
Problem solved.
I mispelled `baseURL` in the yaml file as `baseUrl`
I was the problem.
